src/app/common/hooks/balance/use-stx-available-balance.ts

```diff
@@ -0,0 +1,23 @@
+import { useMemo } from 'react';
+
+import { createMoney } from '@shared/models/money.model';
+
+import { subtractMoney } from '@app/common/money/format-money';
+import { useCurrentStacksAccountAnchoredBalances } from '@app/query/stacks/balance/stx-balance.hooks';
+import { useCurrentAccountMempoolTransactionsBalance } from '@app/query/stacks/mempool/mempool.hooks';
+import { useCurrentAccountMicroblockTransactionsBalance } from '@app/query/stacks/microblock/microblock.hooks';
+
+export function useStxAvailableBalance() {
```

Comment:
It is getting confusing to me in the code about what `availableStx` balance means? If you are going to reuse, and now subtract out pending/microblock txs, should we rename the previous `availableStx` on the balance to something like `unlockedStx`? Just a thought bc I had to lookup and remind myself what was actually different about `balances?.stx.availableStx`. Or, we could rename this hook to something descriptive like, `useStxAvailableBalanceMinusPendingAndMicroblockTxs`? 😆

Reply:
not sure at all, need some help with naming here @kyranjamie @markmhx

Reply:
Agreed naming is getting confused here. Let's sort out in this doc, which also touches on a possible implementation change for `.availableStx` here specifically https://www.notion.so/Balances-UX-905c93b633b84e05a1428c1a9087ab82